packages/manager/modules/web-paas/src/details/service/service.html

```diff
@@ -1,99 +1,155 @@
+                        ></span>
+                    </oui-tile-description>
+                </oui-tile-definition>
+                <oui-tile-definition
+                    data-term="{{ ::'web_paas_service_project_plan_type' | translate }}"
+                    data-description="{{:: $ctrl.project.offerName }}"
+                >
+                </oui-tile-definition>
+                <oui-tile-button data-on-click="$ctrl.goToChangeOffer()">
+                    <span
+                        data-translate="web_paas_service_project_upgrade"
+                    ></span>
+                </oui-tile-button>
+                <oui-tile-definition
+                    data-term="{{:: 'web_paas_service_project_region' | translate }}"
+                    data-description="{{:: $ctrl.CucRegionService.getTranslatedMicroRegion($ctrl.project.getRegion()) }}"
+                ></oui-tile-definition>
+            </oui-tile>
+        </div>
+        <!-- configuration tile -->
+        <div class="col-md-4">
+            <oui-tile
+                data-heading="{{:: 'web_paas_service_project_configuration' | translate }}"
+                class="h-100"
             >
-                <oui-tile-description>
+                <oui-tile-definition
+                    data-term="{{:: 'web_paas_service_project_environment' | translate }}"
+                    data-description="{{:: $ctrl.project.selectedPlan.vcpuConfig }}"
+                ></oui-tile-definition>
+                <oui-tile-button data-on-click="$ctrl.goToChangeOffer(true)">
+                    <span
+                        data-translate="web_paas_service_project_cpu_upgrade"
+                    ></span>
+                </oui-tile-button>
+                <oui-tile-definition
+                    data-term="{{:: 'web_paas_service_project_storage_title' | translate }}"
+                    data-description="{{:: 'web_paas_service_project_storage' | translate: { storageSize: $ctrl.project.getTotalStorage() } }}"
+                >
+                </oui-tile-definition>
+                <oui-tile-button
+                    data-on-click="$ctrl.goToAddAddon('additional-storage')"
+                >
                     <span
-                        data-translate="web_paas_service_project_user_licence_available"
-                        data-translate-values="{ numLicenceAvailable: 0 }"
-                    >
-                    </span>
-                </oui-tile-description>
-            </oui-tile-definition>
-        </oui-tile>
+                        data-translate="web_paas_service_project_storage_upgrade"
+                    ></span>
+                </oui-tile-button>
+                <oui-tile-definition
+                    data-term="{{:: 'web_paas_service_project_user_licences_title' | translate }}"
+                >
+                    <oui-tile-description>
+                        <oui-tile-description>
+                            <p
```

Comment:
Looks like you are using this same status code in multiple places. If so, please extract it into a reusable component.